recipes/bamnostic/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "bamnostic" %}
+{% set version = "0.4.2b9" %}
+{% set sha256 = "a205563e4c9fed1deca50dfb096b59387bc98ec656c5b97327a3d3c119ec5db5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
This is failing since it looks like only a `.whl` is provided on pypi:

https://pypi.org/project/bamnostic/#files

There are no releases on github to grab the source from. Perhaps engage the maintainer in adding a source distribution? My sense from comments on another recent PR is that we really prefer to build these from source.

Reply:
Yes, apologies, I should have written more clearly that this PR is designed to show the package maintainer how to submit a `conda-forge` recipe. I met him at PyCon, and he was angling to build the conda-forge package, so I decided to show him on the spot, and have him added as a recipe maintainer. (I figured with regro bot, maintenance is going to be much easier nowadays.)

Reply:
I've cc'd @betteridiot because he will go back and update the PyPI release with a tarball, and then update the version number and file hash.